### PR TITLE
Lint import aliases

### PR DESCRIPTION
**Pull request by agouin**
- Update linter to be more strict around import aliases 
- introduce [revive](https://revive.run/docs)
- handle revive linter errors

### CODE REVIEW

File: signer/config_test.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestNodes(t *testing.T) {
+func Nodes(t *testing.T) {
```
Comment (mark-rushakoff) [inline signer/config_test.go:16]: Seems wrong -- this will no longer be run as part of `go test`.

File: signer/grpc_server.go [hunk 2]
```diff
@@ -94,8 +94,8 @@ func (rpc *GRPCServer) TransferLeadership(
 }
 
 func (rpc *GRPCServer) GetLeader(
-	ctx context.Context,
-	req *proto.CosignerGRPCGetLeaderRequest,
+	_ context.Context,
+	_ *proto.CosignerGRPCGetLeaderRequest,
```
Comment (mark-rushakoff) [inline signer/grpc_server.go:98]: When every parameter is `_`, you can omit the names altogether: `GetLeader(context.Context, *proto.CosignerGRPCGetLeaderRequest) (...) { }`.

**Review by mark-rushakoff — APPROVED** [review]
Looks straightforward. I think the search and replace caused one test to become not a test anymore, so that should probably be fixed.

**Review by DavidNix — APPROVED** [review]

File: signer/config_test.go (hunk 1 above)
Comment (agouin) [reply signer/config_test.go]: good catch!

**Pull request merged**